DEVELOPMENT.md

```diff
@@ -0,0 +1,331 @@
+<!-- spell-checker:ignore reimplementing toybox RUNTEST CARGOFLAGS nextest prereq autopoint gettext texinfo automake findutils shellenv libexec gnubin -->
```

Comment:
```suggestion
<!-- spell-checker:ignore reimplementing toybox RUNTEST CARGOFLAGS nextest prereq autopoint gettext texinfo automake findutils shellenv libexec gnubin toolchains -->
```

Reply:
Thanks! If you do not mind me asking - what are you using for spell checker? The VSCode spellchecker extension did not catch this, even though it is enabled for markdown files by default